README.md

```diff
@@ -91,6 +91,10 @@ To https://github.com/git-lfs/git-lfs-test
    67fcf6a..47b2002  master -> master
 ```
 
+## Limitations
+
+The current single file size limit for GitHub is 2GB.
```

Comment:
I like that. Maybe extend it with `GitHub and GitHub Enterprise`? 

@bturner Do you know if there is a BitBucket/Stash limitation?
@dosire Do you know if there is a the GitLab limitation?

Reply:
@larsxschneider I can't speak for Bitbucket Cloud (bitbucket.org), but Bitbucket Server/Stash, at least as of 5.8, does not impose any limits on the size of individual files; it only monitors how much free space is available and ensures users can't completely fill the disk. I can't say that that will never change (there is an open feature request for quotas, for example), but as of 5.8 (the current release) there are no limits.

I'd wager there _are_ limits for Bitbucket Cloud. Perhaps @erikvanzijst knows what they are.

Reply:
AFAIK, there are no limitations for self-hosted GitLab (CE, probably the same for EE). I don't know about `gitlab.com`.

As for GitHub, there is also a monthly bandwidth limit for free repositories. No idea for private or Enterprise deployments.